pcdsdevices/utils.py

```diff
@@ -263,8 +269,69 @@ def get_status_float(status_info, *keys, default_value='N/A', precision=4):
     """
     try:
         value = reduce(operator.getitem, keys, status_info)
-        if isinstance(value, float):
-            return ('{:.%df}' % precision).format(value)
+        if isinstance(value, (int, float)):
+            value = float(value) * scale
+            return ('{:.%d%s}' % (precision, format)).format(value)
```

Comment:
Format strings in Python can get gnarly.

This annoyingly does not include the option for showing "+0.00" which would come from: `'{:+.2f}'.format(0.)` (note the placement of the plus). I ran into this on the *first* failed status print from #740. I'd hate to add another argument here (`include_plus_sign=True` and not `format_before_the_dot`). Any ideas?

Reply:
Oh that's really annoying, the formatting a formatter here gets weird.

I don't see any good alternatives to adding a new argument, and having a plus sign seems like a common enough want to include here. Maybe we can have different behavior if you have a `.` in the precision, e.g. `precision="+.2"` could have the behavior you want? Seems awkward still.

Do we want the new argument to be so specific to this case? I think it's fine, but that's because I can't imagine other special cases off the top of my head right now.

Reply:
Hmm, yeah, I hadn't considered allowing `precision` to be a string with that option. Even so, I didn't remember this weird `+` syntax and would hope that users wouldn't have to either. 

Given that the function is predicated on the value being a float, perhaps the boolean arg is the way to go. I'll propose that in the follow-up PR and see if anyone has any thoughts at that point.